src/styles/utils/mod.scss

```diff
@@ -0,0 +1,27 @@
+@for $day from 1 through 31 {
+  .calendar__day:nth-child(#{$day})::before {
+    content: "#{$day}";
+  }
+}
+
+@each $day in mon, tue, wed, thu, fri, sat, sun {
+  .calendar--start-day-#{$day} {
```

Comment:
you create rule but you do not use this class

Reply:
> you create rule but you do not use this class

Not fixed comments without any reply 🤷‍♂️

Reply:
< div  class =" calendar calendar--start-day-fri calendar--month-length-28 " >

Reply:
I use it . Why not?